tests/redirects/map_globalconf.py

```diff
@@ -984,7 +985,8 @@
 
     # from mcom-tests
     url_test('/firefox/', '/en-US/firefox/'),
-    url_test('/en-US/firefox/', '/firefox/new/'),
+    # PMAC: changed locale below from en-US to fr, but test isn't passing
```

Comment:
It will pass when you delete that redirects from `redirects.py`. Also you should keep the `en-US` test to verify the new destination.